docker-app/qfieldcloud/core/admin.py

```diff
@@ -1117,7 +1118,7 @@ def has_module_permission(self, request: HttpRequest) -> bool:
         return False
 
 
-class LogEntryAdmin(ModelAdminEstimateCountMixin, BaseLogEntryAdmin):
+class LogEntryAdmin(QFieldCloudModelAdmin, BaseLogEntryAdmin):
```

Comment:
I am afraid this might cause troubles. We do have `QFieldCloudModelAdmin` that inherits `admin.ModelAdmin` and `BaseLogEntryAdmin` which most likely also inherits `admin.ModelAdmin`. I do not know what will be the consequences, but might cause some weird errorrs we better prevent from happening.

What if we combine the previous and the current approach in something like:

```
class ModelAdminEstimateCountMixin(): ...

class QFieldCloudModelAdmin(ModelAdminEstimateCountMixin, admin.ModelAdmin): ...

# pretty much all admin classes extend directly from QFieldCloudModelAdmin
class OrganizationAdmin(QFieldCloudModelAdmin): ...

# extending only the `ModelAdminEstimateCountMixin` to not extend `admin.ModelAdmin` twice.
class LogEntryAdmin(ModelAdminEstimateCountMixin, BaseLogEntryAdmin): ...
```